cmd/serverless-init/main.go

```diff
@@ -33,11 +34,16 @@ func main() {
 		panic("[datadog init process] invalid argument count, did you forget to set CMD ?")
 	}
 
+	cloudService, logConfig, traceAgent, metricAgent := setup()
```

Comment:
could we move that to an else statement?
I know we panic in the `if` but having [1:] without strong check on the size makes me feel worry